### PR TITLE
feat(Loading): add dark mode

### PR DESCRIPTION
**Pull request by slin12**
### Overview

Adds a dark mode version of the Loader component.

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: https://codecademy.atlassian.net/browse/LX-5578
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

Linked monolith PR: https://github.com/codecademy-engineering/Codecademy/pull/29129

The dark mode colors [come from Stephen](https://codecademy.slack.com/archives/CTVEHH2PP/p1643401636466419?thread_ts=1643397497.746919&cid=CTVEHH2PP).

in action: https://www.loom.com/share/760ad9b1c7d34a82a6b7752f588f1c6c

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.51-alpha.e71523.0<br/>@codecademy/gamut-labs@29.3.0-alpha.e71523.0<br/>@codecademy/styleguide@54.1.0-alpha.e71523.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61f80dd69ce76ef70e8ef0a0--gamut-preview.netlify.app>https://61f80dd69ce76ef70e8ef0a0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61f80dd69ce76ef70e8ef0a0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.59-alpha.7f9075.0<br/>@codecademy/gamut-labs@29.3.0-alpha.7f9075.0<br/>@codecademy/styleguide@54.3.0-alpha.7f9075.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62016079324be72c26c05832--gamut-preview.netlify.app>https://62016079324be72c26c05832--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62016079324be72c26c05832>Deploy Logs</a></p>

File: packages/gamut-labs/src/Loading/index.tsx [hunk 1]
```diff
@@ -1,17 +1,21 @@
+import { ColorModes } from '@codecademy/gamut-styles';
 import { useTheme } from '@emotion/react';
 import cx from 'classnames';
 import React from 'react';
 
+import { useCurrentMode } from '../../../gamut-styles/dist';
```
Comment (jakemhiller) [inline packages/gamut-labs/src/Loading/index.tsx:6]: this should be imported from the module itself, `@codecademy/gamut-styles`

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.51-alpha.3f6d9a.0<br/>@codecademy/gamut-labs@29.3.0-alpha.3f6d9a.0<br/>@codecademy/styleguide@54.1.0-alpha.3f6d9a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://620172e83eada94eba9a6a80--gamut-preview.netlify.app>https://620172e83eada94eba9a6a80--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/620172e83eada94eba9a6a80>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.59-alpha.bcc61c.0<br/>@codecademy/gamut-labs@29.3.0-alpha.bcc61c.0<br/>@codecademy/styleguide@54.3.0-alpha.bcc61c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6202a6dcaf895d2e63d7e028--gamut-preview.netlify.app>https://6202a6dcaf895d2e63d7e028--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6202a6dcaf895d2e63d7e028>Deploy Logs</a></p>

**Review by jakemhiller — APPROVED** [review]

**Pull request merged**